MyClass2.cls

```diff
@@ -3,9 +3,14 @@ public class MyClass {
 	public static void foo1() {
 		for (Integer i=0; i<10; i++) {
 			List<SObject> items = [Select Id, Name FROM account LIMIT 10];
+
 		}
 		items = [Select Id, Name FROM account WHERE Name ='abc'];
 		items = [Select Id, Name FROM account WHERE Name ='abc' LIMIT 10];
```

Comment:
The result of this query is not bound. Please add either a `LIMIT` or a `WHERE` statement to the query to limit the number of results.